src/oatpp/network/tcp/server/ConnectionProvider.cpp

```diff
@@ -240,7 +240,7 @@ oatpp::v_io_handle ConnectionProvider::instantiateServer(){
 
 oatpp::v_io_handle ConnectionProvider::instantiateServer(){
 
-  oatpp::v_io_handle serverHandle;
+  oatpp::v_io_handle serverHandle = -1;
```

Comment:
Let's use `INVALID_IO_HANDLE` instead of `-1`

Reply:
done